action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
   anaconda_nightly_upload_token:
     description: 'Token to upload to scientific python org'
     required: true
+  n_latest_uploads:
+    description: 'Number of latest uploads to keep on the package index'
+    required: false
+    default: 5
```

Comment:
Defaulting to 5 as this was a what `matplotlib` agreed to in https://github.com/matplotlib/matplotlib/pull/23349.